### PR TITLE
Task 7!

### PR DESCRIPTION
**Pull request by TitenkovEA**

### CODE REVIEW

File: src/tat/bsu/homework/lesson2/task7/Main.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    /**
+     * Create a string, and checks her on the rules.
+     *
+     * @param args - command line parameters.
+     */
+    public static void main(String[] args) {
+        try {
+            String string = FromKeyboardReader.stringRead();
+            if (new WithoutNumbersRule().checkRule(string)) {
```
Comment (AlexanderSoroka) [inline src/tat/bsu/homework/lesson2/task7/Main.java]: What if you have to add 100 more rules? Will you add if for every new rule?

File: src/tat/bsu/homework/lesson2/task7/rules/Rule.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package tat.bsu.homework.lesson2.task7.rules;
+
+/**
+ * Represents rule, with checking for compliance with this rule.
+ *
+ * @author Eugeny Titenkov
+ * @version 1.00
+ * @since 9 Oct 2016
+ */
+public interface Rule {
```
Comment (AlexanderSoroka) [inline src/tat/bsu/homework/lesson2/task7/rules/Rule.java]: why interface?

Comment (TitenkovEA) [inline src/tat/bsu/homework/lesson2/task7/rules/Rule.java]: because there is nothing common to realize in abstract class.

Comment (TitenkovEA) [inline src/tat/bsu/homework/lesson2/task7/rules/Rule.java]: and it's more flexibly.

**Pull request merged**